doc/getting_started/getting_started.rst

```diff
@@ -72,19 +100,21 @@ To build an example application follow these steps:
 
       $ export ZEPHYR_GCC_VARIANT=zephyr
       $ export ZEPHYR_SDK_INSTALL_DIR=<sdk installation directory>
+      > set ZEPHYR_GCC_VARIANT=zephyr
```

Comment:
Again, I'd prefer two code blocks rather than intermingling OS instructions like this.